### PR TITLE
openresolv: update to 3.12.0.

### PR DESCRIPTION
**Pull request by subsonik**
Tested OK on x86_64-musl.

### CODE REVIEW

File: srcpkgs/openresolv/template [hunk 1]
```diff
@@ -9,8 +9,8 @@ short_desc="Management framework for resolv.conf"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="BSD-2-Clause"
 homepage="http://roy.marples.name/projects/openresolv/home"
```
Comment (ericonr) [inline srcpkgs/openresolv/template:11]:
```suggestion
homepage="https://roy.marples.name/projects/openresolv"
```

**Comment by ericonr** [comment]
Please rebase the commits into a singe one. I think accepting changes from suggestion doesn't work super well.

**Comment by ericonr** [comment]
Forgot to reset revision :/

**Comment by pullmoll** [comment]
All builders are busy so I think you can just push a fix.

**Pull request closed without merging**